projects/observability/src/shared/components/gauge/gauge.component.ts

```diff
@@ -110,7 +110,7 @@ export class GaugeComponent implements OnChanges {
       innerRadius: radius - GaugeComponent.GAUGE_RING_WIDTH,
       outerRadius: radius,
       startAngle: -completeAngle,
-      endAngle: -completeAngle + 2 * (inputData.value / inputData.maxValue) * completeAngle
+      endAngle: -completeAngle + 2 * Math.max(0.05, inputData.value / inputData.maxValue) * completeAngle
```

Comment:
`.05` feels high here, how does it look? It's showing a 5% fill in the empty case - I was thinking a 1% fill would be sufficient.

Reply:
0.01
<img width="662" alt="Screen Shot 2021-01-11 at 9 56 20 PM" src="https://user-images.githubusercontent.com/48863181/104209639-fee15400-5457-11eb-9114-7b031094a147.png">

0.02
<img width="567" alt="Screen Shot 2021-01-11 at 9 58 24 PM" src="https://user-images.githubusercontent.com/48863181/104209834-33eda680-5458-11eb-8bde-e832363906d7.png">

0.05
<img width="568" alt="Screen Shot 2021-01-11 at 9 56 42 PM" src="https://user-images.githubusercontent.com/48863181/104209681-09035280-5458-11eb-950c-608eb1c2f3c8.png">

Reply:
0.05 looks high, as you said. Looks like a real value. 

0.02 seems to look good as compared to 0.01